zoo-css-challenge/style.css

```
@@ -29,10 +31,31 @@ section {
   background-color: #f8f0fb;
 }
 
+#bear1 {
+  border: solid;
+  border-width: 5px;
+  border-color: red;
+}
+
+#bear2 {
+  border:solid;
+  border-color: blue;
+  border-width: 5px;
```

```suggestion
  border: 5px solid blue;
```
Same as my other comment